recipes/staden_io_lib/meta.yaml

```diff
@@ -14,14 +14,17 @@ source:
     - config.patch
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage('staden_io_lib', max_pin='x.x') }}
 
 requirements:
   build:
     - make
     - {{ compiler('c') }}
+    - autoconf
```

Comment:
This should go to the host section isn't it?

Reply:
I think it goes in the `build` section, along with `automake` and `libtool`; so far, I haven't seen a recipe with these tools in the `host` section.